docs/paper/paper.bib

```diff
@@ -147,7 +147,7 @@ @article{ambikasaran_joss_2019
   volume    = {4},
   number    = {34},
   pages     = {1167},
-  journal   = {The Journal of Open Source Software},
+  journal   = {J. Open Source Softw.},
```

Comment:
I wonder whether this should be JOSS?

Reply:
This is JOSS. The abbrevation was suggested by Wikipedia.

Reply:
I just asked @danielskatz how to abbreviate this journal.

Reply:
Good idea.

as a side note: the abbrevation "J. Open Source Softw." is according to the ISO 4 standard: https://paperpile.com/n/journal-of-open-source-software-abbreviation/